sonaranalyzer-dotnet/its/update-expected.ps1

```diff
@@ -10,6 +19,19 @@ try {
         Write-Error "no 'actual' folder. Have you run this script twice after tests? Have you run the test?"
     }
 
+    if ($project) {
+        Write-Host "Will update issues only for: $project"
+        $projectPath = ".\expected\${project}"
+        if (Test-Path $projectPath) {
+            Remove-Item $projectPath  -Recurse -Force
+        }
+        Copy-Item -Path ".\actual\${project}" -Destination  ".\expected" -Force -Recurse
+        if (Test-Path .\diff) {
+            Remove-Item .\diff -Recurse -Force
+        }
+        return
+    }
```

Comment:
We could extract 2 methods: one which updates all projects and one which updates just the specified one and call these inside the try, after validation.